.github/workflows/R-CMD-check.yaml

```diff
@@ -2,9 +2,9 @@
 # Need help debugging build failures? Start at https://github.com/r-lib/actions#where-to-find-help
 on:
   push:
-    branches: [main, master]
+    branches: [main, master, dev]
```

Comment:
oh, after we get this PR merged... we should rename master to main.    you might have to rename yours locally as well and in your fork.   sorry, meant to do that when we did the repository shuffle...

Reply:
can you change master to main? I do not have permission to do that

Reply:
Yes, will do that now... I believe the PR will automatically get 'moved' to point to the renamed branch as well...

Reply:
Yes, it worked.

Reply:
okay, update this code to remove 'master', and I think it will be doing what you want.